config/nim.cfg

```diff
@@ -220,10 +220,20 @@ llvm_gcc.options.size = "-Os"
 
 # Configuration for the LLVM CLang compiler:
 clang.options.debug = "-g"
+clang.cpp.options.debug = "-g"
 clang.options.always = "-w"
 clang.options.speed = "-O3"
 clang.options.size = "-Os"
 
+@if windows:
+  clang.options.debug = "-g -gcodeview"
+  clang.cpp.options.debug = "-g -gcodeview"
```

Comment:
I think you can omit `-g`, because it's already been specified before this.

Reply:
Nope. Assigning new value overwrites the previous one AFAIK.
The alternative is `clang.options.debug %= "${clang.options.debug} -gcodeview"`